### ISSUE TITLE
value-keyword-case complains on font names

### ISSUE THREAD

**Issue by PhiLhoSoft** [stylelint/stylelint#1132]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Bug, spurious warning.

> Which rule, if any, is this issue related to?

value-keyword-case

> What CSS is needed to reproduce this issue?

```css
$mo-font: OpenSans, Verdana, sans-serif; // In a .scss file
// OR just
font-family: OpenSans, sans-serif;
```

> What stylelint configuration is needed to reproduce this issue?

```yaml
rules:
  value-keyword-case: "lower"
```

> Which version of stylelint are you using?

6.2.1

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint --syntax scss *.scss`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No. Tried with:

```shell
> stylelint foo.css
foo.css
 6:15  ×  Expected "OpenSans" to be "opensans"   value-keyword-case
```

> What did you expect to happen?

No warnings to be flagged, because even if unquoted, font names are not keywords.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged: well, see above!

**Comment by evilebottnawi** [comment]
@PhiLhoSoft https://github.com/stylelint/stylelint/issues/1101

**Comment by PhiLhoSoft** [comment]
Oops, sorry for the noise. I usually check for duplicates before making new issues, but I just forgot this time...
